subxt/src/events/mod.rs

```diff
@@ -31,12 +31,25 @@
 //!
 //! To obtain the events from a given block use `at()`.
 //!
-//! ```rust
-//! let mut events = api.events().subscribe().await?;
+//! ```no_run
+//! # use futures::StreamExt;
+//! # use subxt::{ClientBuilder, DefaultConfig, PolkadotExtrinsicParams};
+//!
+//! #[subxt::subxt(runtime_metadata_path = "../artifacts/polkadot_metadata.scale")]
+//! pub mod polkadot {}
+//!
```

Comment:
Nit: I'd `#` these lines out as well since we hide the lines just below where it's used.